docker-compose.yml

```diff
@@ -79,11 +88,20 @@ services:
       - WEBUI_AUTH=False
       - WEBUI_NAME=deh_ollama_web
       - WEBUI_SECRET_KEY=t0p-s3cr3t
+      - DEFAULT_MODELS=llama3.1:8b-instruct-q3_K_L # Preload llama3.1 model
```

Comment:
Great - I was wondering how to do this so I didn't have to wait for the model load with the initial API request!